src/main/java/org/apache/commons/codec/binary/Base64.java

```diff
@@ -276,6 +381,78 @@ public static byte[] decodeBase64(final String base64String) {
         return new Base64().decode(base64String);
     }
 
+    /**
+     * Decodes standard Base64 data into octets.
+     * <p>
+     * Note: implementation of this method is aligned with the Table 1 of RFC 2045.
+     * </p>
+     * <p>
+     * Note 2 this method skips any unknown or not supported bytes.
+     * </p>
+     *
+     * @param base64Data
+     *            Byte array containing Base64 data
+     * @return Array containing decoded data.
```

Comment:
Mention in `@return` that this is a new array to avoid any possible confusion that the input array may be edited.